packages/react/src/components/AccountOverview/account-overview.scss

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.oxygen-account-overview {
+  padding: 0.8rem 1rem;
+
+  .oxygen-card-header {
+    padding: 1rem 0;
+    padding-bottom: 1.5rem;
+  }
+
+  .oxygen-account-completion-steps-box {
+    padding: 0.5rem;
```

Comment:
Can't we use the default spacing values?

Reply:
Currently we do not have any default spacing values defined in the default theme.